### ISSUE TITLE
Exclude current artwork from "More artwork" section

### ISSUE THREAD

**Issue by reefdog** [OpenTechStrategies/arches-app-exp#89]
On a resource detail page, we have a "More artwork by [artist]" block. This includes all artwork by the artist, inclusive of the current artwork you're viewing.

Preferably, this should only show _other_ artwork by the same artist (and hide entirely if there is none).

But also acceptable if that's too difficult is just to rename this section to "All artwork by [artist]".

**Comment by hminsky2002** [comment]
Excellent write up of the issue, I should've been more explicit but this was actually closed by #86 !

**Comment by reefdog** [comment]
I did later realize you had checked this off your list — for some reason I thought I wasn't seeing this locally, but I am now, so, yay!

That said, I did notice that while it says "More artwork by…" it actually shows _all_ artwork by. As mentioned in the description above, ideally it would exclude the current artwork — failing that, it should be titled "All artwork by…". But I'll bet us geniuses can figure out how to exclude the current one.
